routes.js

```diff
@@ -47,5 +48,9 @@ router.patch('/event-question', auth, (req, res) => eventQuestionController.upda
 router.post('/event-question/response', auth, (req, res) => eventQuestionController.addResponseToEvent(req, res));
 router.delete('/event-question/response/:eventQuestionId/:responseId', auth, (req, res) => eventQuestionController.deleteResponse(req, res));
 
+// ResponseType routes
+router.get('/response-type', (req, resp) => responseTypeController.getAll(req, resp));
+router.get('/response-type/:id', (req, resp) => responseTypeController.getByValue(req,resp));
```

Comment:
I'm not sure if these methods are (super)admin specific. or not. Either way you should at least the `auth `middleware.